src/frontend/src/session.rs

```diff
@@ -184,7 +184,14 @@ impl FrontendEnv {
             meta_config: MetaConfig::default(),
             source_metrics: Arc::new(SourceMetrics::default()),
             creating_streaming_job_tracker: Arc::new(creating_streaming_tracker),
-            compute_runtime: Self::create_compute_runtime(),
+            compute_runtime: Arc::new(BackgroundShutdownRuntime::from(
+                Builder::new_multi_thread()
+                    .worker_threads(4)
```

Comment:
We already generated default value for config, why not use config's value?

Reply:
Fixed, directly use the default value in config